### PR TITLE
Updated pages to include recent events

### PR DESCRIPTION
**Pull request by gdcodes**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,13 +5,13 @@
 custom_js: mailing-list-signup
 
 intro_heading: We're the GU Tech Society
-intro_leadout: Hacker Olympics is coming up soon! </br> Get your ticket below.
+intro_leadout: CDX is coming soon! </br> Get your ticket below.
```
Comment (Edsterr) [inline index.html]: Tickets aren't available, nor are event links, so perhaps best to change it to 'tickets/event available soon' or some such?

File: _data/hacks.yml [hunk 2]
```diff
@@ -1,3 +1,24 @@
+- title: hack2019
+  year: 2019
+  date: 18-20 Oct
+  gallery: hack_2019
+  gallery_size: 0
+  gallery_button_url: https://www.facebook.com/pg/gutechsoc/photos
+  gallery_button_text: Photos will be available soon
+  sponsors:
```
Comment (Edsterr) [inline _data/hacks.yml]: IBM needs to be added

File: _data/hacks.yml [hunk 3]
```diff
@@ -1,3 +1,24 @@
...
+       - guts
+       - school
+       - jpm
+       - ms
+       - skyscanner
+       - craneware
+       - tennents
+       - metix-medical
+       - twilo
```
Comment (Edsterr) [inline _data/hacks.yml]: Typo, should be Twilio. Typo occurs in a few places in the repo

File: index.html (hunk 1 above)
Comment (gdcodes) [reply index.html]: I prevented it from displaying on the website by commenting the line responsible i.e.  but I see it's not the most readable solution. I have now updated the intro_leadout as suggested. Thank you for all the comments!

Comment (gdcodes) [reply index.html]: I prevented it from displaying on the website by commenting the line responsible but I see it's not the most readable solution. I have now updated the intro_leadout as suggested. Thank you for all the comments!

**Pull request merged**